Core/include/Acts/Fitter/KalmanFitter.hpp

```diff
@@ -67,14 +67,14 @@ struct KalmanFitterOptions {
   KalmanFitterOptions(std::reference_wrapper<const GeometryContext> gctx,
                       std::reference_wrapper<const MagneticFieldContext> mctx,
                       std::reference_wrapper<const CalibrationContext> cctx,
-                      const OutlierFinder& outlierFinder,
+                      const OutlierFinder& outlierFinder_,
                       const Surface* rSurface = nullptr,
                       bool mScattering = true, bool eLoss = true,
                       bool bwdFiltering = false)
       : geoContext(gctx),
         magFieldContext(mctx),
         calibrationContext(cctx),
-        outlierFinder(outlierFinder),
+        outlierFinder(outlierFinder_),
```

Comment:
That's weird it did even work. 

Can we make some sort of a naming convention for these cases?

Reply:
Normally this gives a warning but for unknown reasons this is currently suppressed. These are the typical errors that the changes in #267 uncovered.

For now, I would refrain from adding this to the naming convention. As previously mentioned by @paulgessinger, it is still unclear if this will be the final convention. This works for now, but we might decide otherwise in the future.